orthoinference/src/main/resources/Species.json

```diff
@@ -1,4 +1,50 @@
 {
+	"osat":{
+		"mart_url":"http:\/\/plants.ensembl.org\/biomart\/martservice",
+		"refdb":{
+			"url":"http:\/\/plants.ensembl.org\/Oryza_sativa\/Info\/Index",
+			"ensg_access":"http:\/\/plants.ensembl.org\/Oryza_sativa\/geneview?gene=###ID###&db=core",
+			"dbname":[
+				"ENSEMBL",
+				"Ensembl",
+				"ENSEMBL_Oryza_sativa_PROTEIN"
+			],
+			"access":"http:\/\/plants.ensembl.org\/Oryza_sativa\/Transcript\/ProteinSummary?peptide=###ID###"
+		},
+		"mart_group":"osativa_eg_gene",
+		"name":[
+			"Oryza sativa"
+		],
+		"mart_virtual_schema":"plants_mart",
+		"group":"Fungi\/Plants"
+	},
+	"atha":{
+		"mart_url":"http:\/\/plants.ensembl.org\/biomart\/martservice",
+		"refdb":{
+			"url":"http:\/\/plants.ensembl.org\/Arabidopsis_thaliana",
+			"ensg_access":"http:\/\/plants.ensembl.org\/Arabidopsis_thaliana\/geneview?gene=###ID###&db=core",
+			"dbname":[
+				"ENSEMBL",
+				"Ensembl",
+				"ENSEMBL_Arabidopsis_PROTEIN"
+			],
+			"access":"http:\/\/plants.ensembl.org\/Arabidopsis_thaliana\/Transcript\/ProteinSummary?peptide=###ID###"
+		},
+		"alt_refdb":{
+			"url":"http:\/\/www.arabidopsis.org\/index.jsp",
+			"alt_id":"-TAIR-G",
+			"access":"http:\/\/www.arabidopsis.org\/servlets\/TairObject?type=locus&name=###ID###",
+			"dbname":[
+				"TAIR"
+			]
+		},
+		"mart_group":"athaliana_eg_gene",
+		"name":[
+			"Arabidopsis thaliana"
+		],
+		"mart_virtual_schema":"plants_mart",
+		"group":"Fungi\/Plants"
+	},
```

Comment:
@cookersjs If we add rice and arabidopsis in the Species.json file, will this cause them to be inferred for human orthoinference or will they be ignored because they aren't included in the shell script list of target species?

Reply:
Yea the bash script is the determinant of which species is projected to so we won't need to worry about it

Reply:
This file should have a separate version in the PR fork. It will have 80-100 entries!

Reply:
The species file is a config variable so we could always have a 'ReactomeSpecies.json' and a 'PlantReactomeSpecies.json' that still exists on the main branch